### PR TITLE
Load showdown.js dynamically, only when app documentation is actually opened

### PR DESCRIPTION
**Pull request by ianmcorvidae**
Small PR:

This is a somewhat large library that's only relevant in one very specific situation, when app documentation is loaded. This makes it so it's only loaded when needed.

### CODE REVIEW

File: src/components/apps/details/AppDoc.js [hunk 1]
```diff
@@ -77,13 +76,22 @@ function Documentation(props) {
         baseId,
     } = props;
 
+    const [htmlDocumentation, setHtmlDocumentation] = useState("");
```
Comment (ianmcorvidae) [inline src/components/apps/details/AppDoc.js:79]: This starting state is pretty much my main question -- should we have something inforrmative here, or not worry about it?

File: src/components/apps/details/AppDoc.js [hunk 2]
```diff
@@ -77,13 +76,22 @@ function Documentation(props) {
...
+            const showdown = (await import("showdown")).default;
+            const converter = new showdown.Converter();
+            converter.setFlavor("github");
+            setHtmlDocumentation(
+                sanitizeHtml(converter.makeHtml(documentation))
+            );
+        };
+
+        regenerateMarkdown(documentation);
+    }, [documentation]);
```
Comment (sri2k1us) [inline src/components/apps/details/AppDoc.js:94]: 👍

File: src/components/apps/details/AppDoc.js (hunk 1 above)
Comment (sri2k1us) [reply src/components/apps/details/AppDoc.js:79]: I wouldn't worry too much about it give the doc end-point responds really fast.

**Review by sri2k1us — APPROVED** [review]
LGTM :shipit:

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

**Comment by ianmcorvidae** [comment]
Thanks for the reviews! I'll go ahead and merge this one and then go about fixing the other one later today.

**Pull request merged**